app/testOnly/controllers/OptOutTestDataController.scala

```diff
@@ -45,7 +43,8 @@ class OptOutTestDataController @Inject()(
                                           implicit val dateService: DateServiceInterface,
                                           implicit val mcc: MessagesControllerComponents,
                                           val itvcErrorHandler: ItvcErrorHandler,
-                                          implicit val itvcErrorHandlerAgent: AgentItvcErrorHandler
+                                          implicit val itvcErrorHandlerAgent: AgentItvcErrorHandler,
```

Comment:
I don't think this file in anyway related to this change, if we can revert change in this file

Reply:
the pr bot had an issue with the global execution context used here, so i thought it was slightly related